README.md

```diff
@@ -67,17 +67,57 @@ $ npm install --save-dev babel-plugin-react-generate-property
 {
   "env": {
     "development": {
-      "plugins": [["babel-plugin-react-generate-property", { "customProperty": "data-dev", "dirLevel": 2, "slashChar": "\" }]]
+      "plugins": [
+        [
+          "babel-plugin-react-generate-property", {
+            "customProperty": "data-dev",
+            "dirLevel": 2,
+            "slashChar": "\",
+            "addModuleClassNames": true,
+            "prefix": "myPrefix",
+            "firstChildOnly": false
+          }
+        ]
+      ]
     }
   }
 }
 ```
-**customProperty** Use this to configure which property to add to open tags. By default it would be "data-id".
+**customProperty**: Use this to configure which property to add to open tags. By default it would be "data-id".
 
-**dirLevel** How many levels of the file directory do you want to use for the property value. If you use more, the generated value is more likely to be unique, but you will also incur slightly larger builds. Default to 1 (append only the directory where the target file is located in)
+**dirLevel**: How many levels of the file directory do you want to use for the property value. If you use more, the generated value is more likely to be unique, but you will also incur slightly larger builds. Default to 1 (append only the directory where the target file is located in)
 
-**slashChar** Default to "/", if you are on Windows, use "\".
+**omitFileName**  (default: `false`): In case you want to omit filename in data-attr
 
+**slashChar**: Default to "/", if you are on Windows, use "\".
+
+**addModuleClassNames**  (default: `false`) : In case you use css-modules and want to add className to data-attribute:
+
+#### Before
+```jsx
+// src/client/Components/Common/Header.jsx
+<View>
+  <div className={s.wrapper}>...</div>
+</View>
+```
+
+#### Generated
+```jsx
+// using option (addModuleClassNames: true)
+<View data-test="Common_Header_View">
+  <div className={s.wrapper} data-test="Common_Header_div_wrapper">...</div>
+</View>
+```
+
+This does not work for basic string classNames
+
+**prefix**: Add custom prefix to data-attr
+
+**firstChildOnly**: Adds test attrs only on the root DOM node of every component  (default: `false`) 
```

Comment:
Here's an interesting case, not sure if we want to cover, let's say we have:
```
<>
  <MyButton className="ele-1" />
  <MyButton className="ele-2" />
</>
```
Since we skip Fragments `<></>`, looking at the code, it would only add data-id to the first `MyButton`, not both of them, if I read correctly.
Not sure if it's an issue or expected, we don't use this internally.